examples/service-whoami/README.md

```diff
@@ -8,7 +8,7 @@ There is an implementation each of api-token-based `HubAuthenticated` and OAuth-
 
 1.  Launch JupyterHub and the `whoami` services with
 
-        jupyterhub --ip=127.0.0.1
+        jupyterhub
 
 2.  Visit http://127.0.0.1:8000/services/whoami-oauth
 
```

Comment:
The line below mentions `local-system credentials` but `jupyterhub_config.py` is using the dummy authenticator.

Reply:
Good catch, fixed.